finch.yaml.d/mac.yaml

```diff
@@ -39,3 +39,8 @@ firmware:
 
 video:
   display: "none"
+
+hostResolver:
+  hosts:
+    host.finch.internal: host.lima.internal
+    host.docker.internal: host.lima.internal
```

Comment:
Do we want to have this alias? I understand having the Finch alias but this one seems like it could be a little confusing.

Reply:
Has the team discussed this yet?  Looks like the change is approved and only pending this final decision.

Reply:
Hey @Pballer, looks like @Kern-- just rebased and added an e2e test case for using the new domains. After the tests are passing, we should be good to merge.

Reply:
Think we also need to resolve this comment before merging. @sondavidb are you ok with the "docker" alias for the sake of compatibility?

Reply:
Yeah, it's fine, I'm not particularly opinionated on it. Thanks